src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -642,8 +644,11 @@ function BaseSelectionList<TItem extends ListItem>(
                                 testID="selection-list"
                                 onLayout={onSectionListLayout}
                                 style={(!maxToRenderPerBatch || (shouldHideListOnInitialRender && isInitialSectionListRender)) && styles.opacity0}
+                                ListHeaderComponent={listHeaderContent}
                                 ListFooterComponent={listFooterContent ?? ShowMoreButtonInstance}
-                                ListHeaderComponent={listHeaderContent && listHeaderContent}
+                                ListEmptyComponent={listEmptyContent}
+                                contentContainerStyle={isEmptyList && listEmptyContent ? styles.flexGrow1 : undefined}
```

Comment:
@tienifr That's correct however the style on the `BlockingView` is not enough because the blocking view is rendered inside another view. `contentContainerStyle` sets the style on that wrapping view.

<img width="1029" alt="Screenshot 2024-06-10 at 6 15 38 PM" src="https://github.com/Expensify/App/assets/16493223/b7461eab-6b4b-485e-9e5e-9e9a284f54b0">
<img width="1027" alt="Screenshot 2024-06-10 at 6 16 00 PM" src="https://github.com/Expensify/App/assets/16493223/85a86644-f216-40e8-b895-37b23d4b84ed">